docs/message.md

```diff
@@ -42,7 +42,7 @@ $this->setJsFiles($assetManager->getJsFiles());
     ->headerMessage('Very important')
     ->body('<strong>Holy guacamole!</strong> You should check in on some of those fields below.')
     ->size('is-large')
-    ->options(['class' => 'has-text-justified']) ?>
+    ->withOptions(['class' => 'has-text-justified']) ?>
```

Comment:
Adding `with` to widget methods isn't a good idea. Let's revert these naming changes.

Reply:
done.